httpvfs/main.go

```diff
@@ -16,6 +17,7 @@ var (
 	httpAddr   = flag.String("http", ":7070", "HTTP listen address")
 	storageDir = flag.String("dir", ".", "directory to serve via HTTP (caution: HTTP clients can create/edit/delete/view any files in this dir)")
 	verbose    = flag.Bool("v", false, "verbose output")
+	delay      = flag.Int("delay", 0, "artificial handler response latency, in milliseconds, for simulating slowness")
```

Comment:
@sqs, nice, I wasn't aware of it, thanks for the tip!

P.S. A neat feature would be the ability to "add a reaction" to a comment, similar to how Slack lets you do it. That avoids noise, but lets me express that "hey, your comment was helpful to me!"